doc/source/examples/resolved-cfd-dem/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -0,0 +1,345 @@
+==============================================================================
+Sedimentation of one particle.
+==============================================================================
+
+This example aims to reproduce numerically the results obtained by Ten Cate `et al.` `[1] <https://doi.org/10.1063/1.1512918>`_ for the E4 experience. This experience measures the velocity of the sedimentation of a 1.5 cm particle in a container filled with a viscous fluid. The container is sufficiently small to impact the particle sedimentation.
+
+
+.. warning:: 
+    * This case is heavier than most examples. It can take several hours to run.
+    * This case is design with the upcomming IB_contact_and_stability pull Request.
+    
+
+Features
+----------------------------------
+- Solvers: ``gls_sharp_navier_stokes_3d`` (with Q1-Q1)
+- Transient problem
+- Displays the capability of the resolved CFD-DEM solver for the flow around one particle
+
+Description of the case
+-----------------------
+The E4 experiment consiste in the release of a 1.5 cm particle of Nylon (:math:`\rho_p=0.001120 \frac{kg}{cm^3}`) with it center 12.75 centimeter above the bottom of a 10x16x10 cm container. The viscosity of the fluid is :math:`\mu_f=0.00058 \frac{kg}{s cm}` which is equivalent to :math:`\mu_f=0.058 \frac{N s}{m^2}`. The density of the fluid is :math:`\rho_f=0.000960 \frac{kg}{cm^3}`. The gravity is :math:`g= -981 \frac{cm}{s^2}`. The particle accelerate due to the gravity until it hit the bottom of the container, at which point we stop the simulation.
+
+.. note:: 
+    You will note that we have transformed every length unit into centimeters. The reason is that the particle is very close to 1 cm. Representing the problem in this way helps the linear solver. It avoids extremely small values in the matrix due to the volume of cells being expressed in cm^3 instead of m^3. 
```

Comment:
```suggestion
    You will note that we have transformed every length unit into centimeters. The reason is that the particle's size is very close to 1 cm. Representing the problem in this way helps the linear solver converge. It avoids extremely small values in the matrix due to the volume of cells being expressed in \si{cm^3} instead of \si{m^3}. 
```